### ISSUE TITLE
Feature: Report the number of exceptions included in .nsprc

### ISSUE THREAD

**Issue by mseefeldt** [nodesecurity/nsp#217]
Currently it is possible to include exceptions in .nsprc (which is a great and useful feature).

It would be great if the grunt-task and also nsp report the number of exceptions stored in .nsprc.

The reasoning:
In order to have a build pass currently, i included all issues in .nsprc.

When i approach a release of the project, the exception list should be empty at best. The number of included exception would then be a great reminder that there are still exceptions which are not solved.

**Comment by darkwebdev** [comment]
yeah, otherwise developers will just forget to remove exceptions over time...
